index.html

```diff
@@ -54,8 +54,8 @@
     </pre></section>
 
     <section class="copyright"><pre>
-&copy; 2012-2014 Sam Stephenson, Basecamp
-</pre></section>
+&copy; 2012-2016 Sam Stephenson, Basecamp
```

Comment:
How about using Jekyll to get the current year. 

`&copy; 2012-{{ 'now' | date: "%Y" }}  Sam Stephenson, Basecamp`

Reply:
Well...we are in 2019 already....

Reply:
How the years go by…